gpkit/nomials/nomial_math.py

```diff
@@ -464,26 +342,28 @@ class Posynomial(Signomial):
     """
     def __le__(self, other):
         if isinstance(other, Numbers + (Monomial,)):
-            return Constraint(other, self, oper_ge=True)
+            return PosynomialInequality(self, "<=", other)
         else:
             # fall back on other's __ge__
             return NotImplemented
+        """Support the / operator in Python 3.x"""
+        return self.__div__(other)
```

Comment:
above 2 lines look like they got mixed up in a copy paste -- is it the missing `__div__`? Why didn't this fail a unit test?